fastn-core/src/library2022/processor/user_details.rs

```diff
@@ -5,12 +5,16 @@ pub async fn process(
     doc: &ftd::interpreter::TDoc<'_>,
     req_config: &fastn_core::RequestConfig,
 ) -> ftd::interpreter::Result<ftd::interpreter::Value> {
-    match ud(
-        &req_config.config.ds,
-        req_config.config.get_db_url().await.as_str(),
-        req_config.request.cookie("fastn-sid"),
-    )
-    .await
+    match req_config
+        .config
+        .ds
+        .ud(
+            req_config.config.get_db_url().await.as_str(),
+            &req_config
+                .request
+                .cookie(fastn_core::http::SESSION_COOKIE_NAME),
```

Comment:
This should also use the `session_id()` method.

Reply:
Done. Thanks.